doc/templates.md

```diff
@@ -41,6 +41,25 @@ Default fallback order is the following:
 
 > Bundle fallback order is the instantiation order in `AppKernel`.
 
+### Additional theme paths
+In addition to the convention described above, it is also possible to add arbitrary Twig template directories to a theme
+from configuration. This can be useful when you want to define templates from third-party bundles as part of one of your
+themes, or when upgrading your application in order to use eZ Platform design engine, and that your existing templates
+are not yet following the convention.
+
+```yaml
+ezdesign:
+    design_list:
+        my_design: [my_theme, some_other_theme]
+    template_theme_paths:
+        # FOSUserBundle templates will be part of "my_theme" theme
+        my_theme:
+            - '%kernel.root_dir%/../vendor/friendsofsymfony/user-bundle/Resources/views'
```

Comment:
I was tempted to use `%kernel.project_dir%` but as we don't require Symfony 3.3 yet, I preferred using the old fashion way 😛